docs/src/main/tut/resources_for_learners.md

```diff
@@ -4,6 +4,24 @@ title:  "Resources for Learners"
 section: "resources_for_learners"
 position: 30
 ---
+
+The name *Cats* is a playful shortening of the word *category*. This term derives from
+*Category theory*, a branch of abstract mathematics with far reaching implications
+in Computer Science that
+gained popularity in [Haskell](https://www.haskell.org/). the Cats library implements
+Category theory concepts in Scala, owing a debit to Haskell's pioneering efforts to codify
+Category theory concepts in a language. Cats has many similiarities to Haskell's codification
+of Category theory into a computer language, but with some improvements along with implementation
```

Comment:
Minor: the word "improvements" makes me a bit uneasy here. What about something like "but with some adjustments to better fit Scala and the JVM"?